cpp/include/cudf/column/column_view.hpp

```diff
@@ -288,9 +288,9 @@ class column_view_base {
   column_view_base(data_type type,
                    size_type size,
                    void const* data,
-                   bitmask_type const* null_mask = nullptr,
-                   size_type null_count          = UNKNOWN_NULL_COUNT,
-                   size_type offset              = 0);
+                   bitmask_type const* null_mask,
+                   size_type null_count,
```

Comment:
null_mask and null_count are still documented as optional (same goes for the other two functions).

Reply:
Good catch, thanks!